src/components/layout/be-square/footer.js

```diff
@@ -0,0 +1,92 @@
+} from './style/footer'
+import { localize, Localize } from 'components/localization'
+import Linkedin from 'images/svg/be-square/linkedin.svg'
+import Twitter from 'images/svg/be-square/twitter.svg'
+import Facebook from 'images/svg/be-square/facebook.svg'
+import Instagram from 'images/svg/be-square/instagram.svg'
+
+const query = graphql`
+    query {
+        footer_image: file(relativePath: { eq: "be-square/footer-image.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const BeSquareFooter = () => {
+    const data = useStaticQuery(query)
+    return (
+        <React.Fragment>
+            <Section color="black">
+                <ContentContainer>
+                    <ImageWrapper data={data['footer_image']} alt="Deriv employee" />
+                    <TextContainer>
+                        <Title>{localize('Next intake: July 2021')}</Title>
+                        <TextWrapper
+                            max_width={['510px', '328px']}
+                            font_size={['20px', '16px']}
+                            line_height={['30px', '24px']}
+                        >
+                            <Localize
+                                translate_text="Submit your application before 30 May 2021<0/>If you have any questions, email us at <1>info@besquare.my</1>"
+                                components={[
+                                    <br key={0} />,
+                                    <Link
+                                        font_size={['20px', '16px']}
+                                        line_height={['30px', '24px']}
+                                        href="mailto:info@besquare.my"
+                                        key={1}
+                                    />,
+                                ]}
+                            />
+                        </TextWrapper>
+                        <ButtonWrapper type="submit" secondary="true" to="#">
+                            {localize('Apply now')}
+                        </ButtonWrapper>
+                        <FollowUsContainer>
+                            <TextWrapper
+                                max_width={['68px']}
+                                font_size={['16px']}
+                                line_height={['24px']}
+                            >
+                                {localize('Follow us')}
+                            </TextWrapper>
+                            <img src={Linkedin} alt="Linkedin icon" width="26.67" height="26.67" />
+                            <img src={Facebook} alt="Facebook icon" width="26.67" height="26.67" />
+                            <img
+                                src={Instagram}
+                                alt="Instagram icon"
+                                width="26.67"
+                                height="26.67"
+                            />
+                            <img src={Twitter} alt="Twitter icon" width="26.67" height="26.67" />
```

Comment:
You should also attach the social links for these to be redirected when clicked 😬

Reply:
The links are not ready yet, that's why they are just an icon for now 😄